Codecademy/gamut
# feat(Organisms)

### Overview


Adding Organisms stories to Storybook 8


### PR Checklist

- [ ] Related to JIRA ticket: [GM-918]

#### Testing Instructions



1. Check over the preview link for Organisms (https://674741c1041fd02a689515ea--gamut-preview.netlify.app/?path=/docs/organisms-about--docs) 
2. Click through the stories to see if they render properly 
3. Proofread along the way?
4. ...
5. Profit! 



[GM-918]: https://skillsoftdev.atlassian.net/browse/GM-918?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
